arch/Kconfig

```diff
@@ -178,10 +178,7 @@ config USERSPACE
 
 config PRIVILEGED_STACK_SIZE
 	int "Size of privileged stack"
-	default 512 if MPU_STACK_GUARD || BUILTIN_STACK_GUARD
-	default 512 if COVERAGE_GCOV
-	default 384 if ARC
-	default 256
+	default 1024
```

Comment:
I agree with the rationale, here, but, I see that the priv stack starts getting too big

Reply:
not sure what you mean by too big
The actual minimum value for this, that satisfies maximum depth for all kernel/driver syscalls is unknown at this time.